ff-netF48-server-sdk.csproj

```diff
@@ -52,114 +52,6 @@
     <WCFMetadata Include="Connected Services" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
I have no idea why all of this stuff was here. AFAICT it is completely unnecessary.